.eslintrc.js

```diff
@@ -41,5 +42,17 @@ module.exports = {
             ignoreTemplateLiterals: true,
             ignoreRegExpLiterals: true,
         }],
+        'no-use-before-define': ['error', {
+            functions: true,
+            classes: true,
+            variables: true,
+            allowNamedExports: false,
+        }],
+        'linebreak-style': [
+        'error', 'windows',
+        ],
+        'import/prefer-default-export': [
+            ["off" | "warn" | "error"],
```

Comment:
You should choose one of these options: or `off`, or `warn`, or `error`

Reply:
Updated